nvflare/private/fed/server/job_runner.py

```diff
@@ -328,28 +330,32 @@ def _save_workspace(self, fl_ctx: FLContext):
         shutil.rmtree(run_dir)
 
     def run(self, fl_ctx: FLContext):
+        """Starts job runner."""
         engine = fl_ctx.get_engine()
-
```

Comment:
It is okay to start the thread here.

Reply:
The job_complete_process requires job_manager to be True,
So I think it is better to move the start after the check